django_reference_app/django_ref_config/settings.py

```diff
@@ -80,12 +80,16 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    }
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'kubernetes_django',
```

Comment:
General convention is for the database name to match the application name, or at least be related to it - so here I'd call it `django-reference` or similar.

Also as a general note - this application should never reference kubernetes, or the cluster, as that implies that the app is somehow designed specifically for our hosting environment, which isn't the case.

Reply:
No problem. Changed name in settings.py and deployment.yaml.